### DISCUSSION TITLE
Encountered circular relationship

### DISCUSSION THREAD

**Discussion by cmpich** [geneontology/helpdesk#490] [General]
Lat week's update of the gene ontology obo file caused the following circular relationships between the following two GO terms:

GO:0045282  | plasma membrane succinate dehydrogenase complex              | GO:0045258  | plasma membrane succinate dehydrogenase complex (ubiquinone) | is_a
 GO:0045258  | plasma membrane succinate dehydrogenase complex (ubiquinone) | GO:0045282  | plasma membrane succinate dehydrogenase complex              | part of

on the one hand the first term has a is_a relationship to the second term while 
the second term has a part_of relationship to the first term. 
Our closure logic is choking on this relationship as the two relationships are considered equal. 
To me this relationship does not make sense. Any ideas?

**Comment by cmpich** [comment]
[Term]
id: GO:0045258
name: plasma membrane succinate dehydrogenase complex (ubiquinone)
is_a: GO:0045257 ! succinate dehydrogenase complex (ubiquinone)
is_a: GO:0045282 ! plasma membrane succinate dehydrogenase complex
intersection_of: part_of GO:0005886 ! plasma membrane

[Term]
id: GO:0045282
name: plasma membrane succinate dehydrogenase complex
is_a: GO:0045281 ! succinate dehydrogenase complex
is_a: GO:0098797 ! plasma membrane protein complex
intersection_of: GO:0045281 ! succinate dehydrogenase complex
intersection_of: part_of GO:0005886 ! plasma membrane
relationship: part_of GO:0045258 ! plasma membrane succinate dehydrogenase complex (ubiquinone)
relationship: part_of GO:0045274 ! plasma membrane respiratory chain complex II

**Comment by suzialeksander** [comment]
Noting this is the same as the issue reported in https://github.com/geneontology/helpdesk/issues/489, in the same area of the ontology

**Comment by pgaudet** [comment]
Right, this should be closed as a duplicate (and AFAIK it's fixed)
